src/top_model/main.cpp

```diff
@@ -225,6 +227,20 @@ std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> TOP = std::make_share
     input = fopen("../data/output/abp_output.txt","r");
     output = fopen("../data/output/abp_processed_output.csv","w");
     filter(input,output);
-    
+    fclose(input);
+    fclose(output);
+    output_filter();
     return 0;
+}
+
+void output_filter(){

```

Comment:
Why this function is defined inside main.cpp?

Reply:
Hi, 
Code has been updated in a better way, by removing the output_filter() function from all source files.
Thanks!